docs/api/qiskit-ibm-provider/release-notes.md

```diff
@@ -68,7 +68,7 @@ in_page_toc_max_heading_level: 2
 
 ### Upgrade Notes
 
-*   Job error messages now include the error code. Error codes can be found in [https://quantum-computing.ibm.com/lab/docs/iql/manage/errors](https://quantum-computing.ibm.com/lab/docs/iql/manage/errors).
+*   Job error messages now include the error code. Error codes can be found in [../lab/docs/iql/manage/errors](../lab/docs/iql/manage/errors).
```

Comment:
This is also a bad type of relative link. The site doesn't start with `docs.quantum-computing`.